buid.sbt

```diff
@@ -0,0 +1,47 @@
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.DefaultBuildSettings.targetJvm
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
+
+lazy val appName = "cc-calculator"
+
+lazy val appDependencies : Seq[ModuleID] = ???
+lazy val plugins : Seq[Plugins] = Seq(
+  SbtAutoBuildPlugin,
+  SbtGitVersioning,
+  SbtDistributablesPlugin,
+  SbtArtifactory
+)
+lazy val playSettings : Seq[Setting[_]] = Seq.empty
+
+lazy val scoverageSettings = {
+  Seq(
+    ScoverageKeys.coverageExcludedPackages :=  "<empty>;Reverse.*;.*Routes.*;routes_routing.*;uk.gov.hmrc;config.*;",
+    ScoverageKeys.coverageMinimum := 95,
+    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageHighlighting := true,
+    parallelExecution in Test := false
+  )
+}
+
+lazy val microservice = Project(appName, file("."))
+  .enablePlugins(Seq(play.sbt.PlayScala) ++ plugins : _*)
+  .settings(playSettings ++ scoverageSettings : _*)
+  .settings(PlayKeys.playDefaultPort := 9372)
+  .settings(majorVersion := 1)
+  .settings(publishingSettings : _*)
+  .settings(
+    targetJvm := "jvm-1.8",
+    scalaVersion := "2.11.12",
```

Comment:
The scala version is set as 2.11.12, but the build-job has it as 2.12.x. It's also set in multiple places

Reply:
There were 2 copies of scalaVersion both set to 2.11.12. Removed the other and upgraded to 2.12.10